library/src/main/res/values/strings.xml

```diff
@@ -1,4 +1,13 @@
 <resources>
+    <string name="app_name">GetStreamChatSDK</string>
+
+    <!--Notifacation-->
+    <string name="default_notification_channel_id">stream_GetStreamChat</string>
```

Comment:
All strings should have prefix `stream_`. 
Otherwise we're mixing sdk resources with customer app resources.